ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.6.yaml

```diff
@@ -153,7 +153,7 @@ tests:
     workflow: openshift-e2e-libvirt
 - as: ocp-jenkins-e2e-remote-libvirt-ppc64le
   cluster: build01
-  cron: 0 2 * * 1
+  cron: 0 1 2 1 *
```

Comment:
Ditto

Reply:
hmm. Jan 2nd at 1:00am UTC every year.

Reply:
Jenkins jobs will start at 1:00am UTC on Jan 02, 03, 04, 05 for OCP 4.6, 4.7, 4.8 and 4.9 in that order.